src/components/InfoFooter.vue

```diff
@@ -31,22 +30,25 @@ export default {
   data() {
     return {
         environment: process.env.NODE_ENV,
-        release: "",
         repo: ""
       }
   },
   methods: {
+    getRelease() {
+      if (this.environment != 'development') {
+        return this.repo.slice(51, this.repo.length)
```

Comment:
What is 51 and 38? Guessing its number of letters for something? Maybe they could be declared as a constant or data element to make it clean :)